draft-ietf-taps-interface.md

```diff
@@ -1740,9 +1767,29 @@ The ReceiveError event passes an optional associated MessageContext. This may
 indicate that a Message that was being partially received previously, but had not
 completed, encountered an error and will not be completed.
 
-## Message Receive Context {#receive-context}
+## Message Contexts {#msg-ctx}
+
+A MessageContext object contains properties, see {{message-props}}, and other
+metadata for Messages. Therefore, a MessageContext object can be passed to the
+Send Action and is retuned by each Send action as well as Send and Receive
+related events.
+
+The application can set, see {{message-props}} and query Message Properties using the Message Context.
```

Comment:
I think this is grammatically challenged. PERHAPS it's correct if you add a comma after {{message-props}}, dunno; I'm quite sure it's correct if you use parantheses instead, i.e.: "The application can set (see {{message-props}}) and query..."

Reply:
Rewrote the paragraph to make it more readable